src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -0,0 +1,61 @@
+import _ from 'lodash/fp';
+
+import * as currentOrPastDate from '../../common/schemaform/definitions/currentOrPastDate';
+import * as fullName from '../../common/schemaform/definitions/fullName';
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+import definitions from 'vets-json-schema/dist/definitions.json';
+
+import { relationshipLabels, genderLabels } from '../utils/helpers';
+
+const defaultProps = [
+  'relativeFullName',
+  'relativeSocialSecurityNumber',
+  'relativeDateOfBirth',
+  'gender',
+  'relationship'
+];
+
+export default function applicantInformation(
+  shema,
```

Comment:
Looks like this isn't used (and has a typo)? I think you can use it instead of that definitions.json file.